src/Synchronization/OrderSync.php

```diff
@@ -97,7 +98,7 @@ public function sync($order_id ) {
                 try {
                     $guzzleClient->put("/orders/{$trackmage_order_id}", [
                         'json' => [
-                            'status' => ['name' => $order->get_status()],
+                            'orderStatus' => ['code' => $this->getTrackMageStatus($order)],
```

Comment:
add `title` here. maybe `'orderStatus' => $this->getTrackMageStatus()`, and return ['code', 'title'] in `getTrackMageStatus`.